substrafl/experiment.py

```diff
@@ -365,6 +369,8 @@ def execute_experiment(
             name=name or timestamp,
             metadata=cp_metadata,
         ),
+        auto_batching=True,
```

Comment:
Outside of this PR and this répo but the `auto_batching` name is miss leading. I expected it to automatically compute an optimal batch size, it only enables batching. 
Am I the only one in that case (if no, I'll create an asana card)

Reply:
there is already a card